packages/test-utils/lib/test-utils.js

```diff
@@ -18,7 +18,7 @@ async function setupServer({
   const Adapter = { mongoose: MongooseAdapter, knex: KnexAdapter }[adapterName];
 
   const argGenerator = {
-    mongoose: getMongoMemoryServerConfig,
+    mongoose: async () => await getMongoMemoryServerConfig(),
```

Comment:
If we decided we don't need the DBName we can run this without async?

Reply:
yep, this change can be reverted.